src/System.Net.Http/src/System/Net/Http/Managed/HttpEnvironmentProxy.cs

```diff
@@ -36,7 +36,7 @@ public NetworkCredential GetCredential(Uri uri, string authType)
                    uri.Equals(_httpsProxy) ? _httpsCred : null;
         }
 
-        public static HttpEnvironmentProxyCredentials TryToCreate(Uri httpProxy, Uri httpsProxy)
+        public static HttpEnvironmentProxyCredentials TryCreate(Uri httpProxy, Uri httpsProxy)
```

Comment:
These TryCreate methods are currently not exposed publicly, so it's not a big deal for now.  Just FYI that the "Try" pattern normally returns `bool` and passes the resulting value as an `out`, e.g.
```C#
public static bool TryCreate(Uri httpProxy, Uri httpsProxy, out HttpEnvironmentProxyCredentials credentials) { ... }
```
Again, you don't need to change it now, just highlighting the difference.